site/schemas/change-password.schema.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from "zod";
+import { setZodDefaultError } from "../utils";
+
+export const changePasswordSchema = z.object({
+    currentPassword: z.string(setZodDefaultError("Enter your current password")).min(1),
+    newPassword: z.string().min(8, {
```

Comment:
Might be worth making the password length a shared constant since we'll need it on the register page as well

Reply:
Customising error message will not be possible if its used as a shared constant